packages/svelte/test/performance.test.ts

```diff
@@ -0,0 +1,191 @@
+import { Scope } from '@sentry/hub';
+import { act, render } from '@testing-library/svelte';
+
+import DummyComponent from './components/Dummy.svelte';
+
+let returnUndefinedTransaction = false;
+
+let testTransaction: { spans: any[]; startChild: jest.Mock; finish: jest.Mock } = {
+  spans: [],
+  startChild: jest.fn(),
+  finish: jest.fn(),
+};
+let testUpdateSpan = { finish: jest.fn() };
+let testInitSpan: any = {
+  transaction: testTransaction,
+  finish: jest.fn(),
+  startChild: jest.fn(),
+};
+
+jest.mock('@sentry/hub', () => {
+  const original = jest.requireActual('@sentry/hub');
+  return {
+    ...original,
+    getCurrentHub(): {
+      getScope(): Scope;
+    } {
+      return {
+        getScope(): any {
+          return {
+            getTransaction: () => {
+              return returnUndefinedTransaction ? undefined : testTransaction;
+            },
+          };
+        },
+      };
+    },
+  };
+});
+
+describe('Sentry.trackComponent()', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    testTransaction.spans = [];
+
+    testTransaction.startChild.mockImplementation(spanCtx => {
```

Comment:
Sometimes, Jest is a mystery to me. Apparently, `mockImplementation` (or `mockReturnValue`) must be called inside a `describe` or `it`/`test` function, while I can create empty `jest.fn()` mocks anywhere. I find this a little weird but whatever